var/spack/repos/builtin/packages/osqp/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Osqp(CMakePackage):
+    """
+    The OSQP (Operator Splitting Quadratic Program) solver is a numerical
+    optimization package for solving problems in the form:
+     minimize        0.5 x' P x + q' x, and
+     subject to      l <= A x <= u,
+    where `x in R^n` is the optimization variable.
```

Comment:
This may not format well in `spack info osqp`, you'll have to try it out and see.